CONTRIBUTING.md

```diff
@@ -342,6 +342,23 @@ If you are a Guru, try to restrain yourself, only do magic when absolutely neces
           // Log message already printed above
       }
       ```
+    * Deliberate switch-case fall through should be explicitly indicated with a specific comment:
```

Comment:
I think this can be an item in the top-level list, not inside *Conditionals*.

Reply:
My reasoning is that switch is technically a conditional statement so thats why i put it there. Do you think its more appropriate outside of contidionals as its own thing?

Reply:
You are right, but the fall-through is specific to switch(), it has nothing to do with syntax and style for (other) conditionals.

Reply:
alright, should i name the list item `* switch-case deliberate fallthrough`?

Reply:
Yeah, that's fine.